build.env

```diff
@@ -5,4 +5,4 @@ PROJECT=bb
 MFA_ENABLED=false
 
 # Terraform
-TERRAFORM_IMAGE_TAG=1.3.5-0.1.17
+TERRAFORM_IMAGE_TAG=1.3.5-0.2.0
```

Comment:
@exequielrafaela Isn't that image only compatible with RC `1.12.5rc1`?

Reply:
@diego-ojeda-binbash You're right. I'll discard this change for the moment. Thanks!

Reply:
@diego-ojeda-binbash the latest release covers this. I'll move forward and merge it now.